bin/bootstrax

```diff
@@ -1173,7 +1188,9 @@ def process_run(rd, send_heartbeats=args.production):
                        'strax_fragment_payload_bytes')
             for conf in to_read:
                 if conf not in dq_conf:
-                    log_warning(f'{conf} not in rundoc for {run_id}! Using default values.')
+                    log_warning(f'{conf} not in rundoc for {run_id}! Using default values.',
+                                priority='info',
+                                run_id=run_id)
```

Comment:
Rather than potentially log several messages in this for-loop, it might be cleaner to accumulate them and log outside the loop.